html/statbrowser.js

```diff
@@ -369,6 +371,13 @@ function draw_status() {
 			document
 				.getElementById("statcontent")
 				.appendChild(document.createElement("br"));
+		} else if (
+			// hardcoded for testing purposes .includes() seems to be breaking things for some reason.
```

Comment:
comment?

Reply:
to clarify: this is final code, so either it's hardcoded for good (and not testing purposes) or this is still an outstanding problem that needs to be fixed